src/java.desktop/share/classes/java/awt/doc-files/Modality.html

```diff
@@ -356,8 +349,8 @@ <h2>Compatibility</h2>
       calls: <code>Dialog.setModal(true)</code>,
       <code>Dialog(owner, true)</code>, etc. Prior to JDK 6
       the default type was toolkit-modal,
-      but the only distinction between application- and toolkit-modality is for
-      applets and applications launched from Java Web Start.
+      and now with single application per-VM there is no
+      distinction between application- and toolkit-modality
```

Comment:
```suggestion
      distinction between application- and toolkit-modality.
```
End the sentence with a full stop?